README.md

```diff
@@ -1,32 +1,52 @@
 # Ably Documentation Overview
 
-[Ably](https://www.ably.io) is a hugely scalable, superfast and secure hosted realtime messaging service for web-enabled devices.
+[Ably](https://www.ably.io) is a scalable, fast, and secure hosted realtime messaging service for web-enabled devices.
 
-This Git repository contains most of the [Ably API documentation](https://www.ably.io/documentation) that resides at <https://www.ably.io/documentation> in [Textile](redcloth.org/textile) format.  [Nanoc](http://nanoc.stoneship.org/) is used to build the static site from the `master` branch.
+The static site generated from this repository is hosted at <http://docs.ably.io> and deployed automatically when the `master` branch is updated.  
 
-The Ably documentation was intentionally created as a public repository using a simple text based markup language so that:
+We frequently publish updates from this repository so is typically more up to date than the official Ably documentation which can be found at <https://www.ably.io/documentation>
 
-* It can be maintained by Ably staff and developers.
-* 3rd parties can fork the repository, and submit pull requests to improve the documentation.  We welcome the community's suggestions and input.
-* This documentation is later merged upstream to the [Ably primary website documentation](https://www.ably.io/documentation) with each website release.
+This Git repository uses the [Textile](redcloth.org/textile) format with [Nanoc](http://nanoc.stoneship.org/) used to build the static site from the `master` branch.
 
-## Viewing the documentation
+## Running locally
 
-The static site generated from this documentation repository is hosted at <http://docs.ably.io> and deployed automatically when the `master` branch is updated.  We frequently publish updates from this repository so is typically more up to date than the [official Ably API documentation](https://www.ably.io/documentation).
+1. Clone a local version of the repo, or a local version of your forked repo.
+3. Checkout the master branch.
```

Comment:
Numbering goes 1-3-4 (Markdown makes it look correct when rendered, but probably better to either make the numbering match correctly, or go for the old style 1-1-1 for consistency).